src/inmanta/app.py

```diff
@@ -207,6 +207,44 @@ async def safe_shutdown_wrapper(shutdown_function: Callable[[], Coroutine[Any, A
         IOLoop.current().stop()
 
 
+class FeatureFlags:
```

Comment:
```suggestion
class ExperimentalFeatureFlags:
```
It looks like this class is solely for experimental features (implementation of `add_arguments`. The naming could make this clearer.

Reply:
done